src/js/setup.js

```diff
@@ -89,10 +96,10 @@ function autoSetupTimeout(wait, vjs) {
     videojs = vjs;
   }
 
-  setTimeout(autoSetup, wait);
+  window.setTimeout(autoSetup, wait);
```

Comment:
True, but it's consistent with the rest of the codebase where we are using `window.setTimeout`, which will work in Node because we're using the `global` package.